src/pages/Delivery/Delivery.styled.jsx

```diff
@@ -47,10 +50,9 @@ const HeaderText = styled.div`
   }
 `
 const ChatBtnContainer = styled.div`
-  width: 28.33vw;
   height: auto;
-  display: flex;
-  justify-content: flex-end;
+  position: absolute;
```

Comment:
if ChatBtnContainer has position absolute, the father should have position relative (the navbar)